### ISSUE TITLE
Welcome/Authenticated pop up is not showing.

### ISSUE THREAD

**Issue by bionics07** [playgameservices/play-games-plugin-for-unity#3130]
This form is for reporting Unity Plugin issues only. To report an issue with the Play Games Service (non-SDK related), check [Google Play Games Services Support](https://developers.google.com/games/services/support).
*Once you've read this section and determined that your issue is appropriate for this repository, please delete this section.*

**BUG**
In my game i can authenticate with `PlayGamesPlatform.Instance.Authenticate(ProcessAuthentication);` but i can't see the pop up informing that to me. I was using this package before and now I'm trying to update to **version 11.01**.

**Expected behavior**
I want to see the pop up when i authenticate.

**Versions**
- Unity version: 2020.3.22f1
- Google Play Games Plugin for Unity version: 11.01

**Additional context**
This is called in a loading step of the game.
```
        base.SignIn(onSuccess, onFailure, silent);
        PlayGamesPlatform.DebugLogEnabled = true;
        PlayGamesPlatform.Activate();
        if (silent)
        {
            //PlayGamesPlatform.Instance.Authenticate(ProcessAuthentication);
            Social.localUser.Authenticate(OnSignInResult);
        }
        else
        {
            PlayGamesPlatform.Instance.ManuallyAuthenticate(ProcessAuthentication);
        }
        void ProcessAuthentication(SignInStatus status)
        {
            if (status == SignInStatus.Success)
            {
                // Continue with Play Games Services
                OnSignInResult(true);
            }
            else
            {
                OnSignInResult(false);
                // Disable your integration with Play Games Services or show a login button
                // to ask users to sign-in. Clicking it should call
                // PlayGamesPlatform.Instance.ManuallyAuthenticate(ProcessAuthentication).
            }
        }
```

**Comment by vailshnast** [comment]
Did you manage to fix this or you reverted back to previous version?

**Comment by berkaytndgn** [comment]
Hello, I am having the same problem. The authorization process is completed in the background and getting SignInStatus.Success, but cannot see any welcome popup.

**Comment by bionics07** [comment]
> Did you manage to fix this or you reverted back to previous version?

@vailshnast i tried to fix and search for something about but i hadn't find some fix, so i just did a rollback for previus version.

**Comment by berkaytndgn** [comment]
> Still actual. Is there any stable version that has to be used?

v10 is currently stable. I also rolled back to v10.

**Comment by BlackCatKPI** [comment]
You're talking about v. 0.10.14, right?

**Comment by berkaytndgn** [comment]
> You're talking about v. 0.10.14, right?

Exactly.

**Comment by ksvslk** [comment]
Up!

**Comment by Kenorbrook** [comment]
Hello, I am having the same problem. The authorization process is completed in the background and getting SignInStatus.Success, but cannot see any welcome popup.

**Comment by userqt** [comment]
@ozdemir08 , could you explain a bit what does it mean that the roll-out mentioned above is in progress? Which app (Play Services App) is that exactly? Do we have to wait for some days until it is updated?

**Comment by userqt** [comment]
Looks like this issue and project in general receives quite low response rates.

Regarding the native authorization pop up not showing, I found something new, which might help someone else who stumbles in here. If you had already created a Google Play Games profile account inside the app, to check if native UI shows in general you can delete the account by going into the app -> click the three dots -> Delete Play Games account & data. You can do this only if you have a test account created, otherwise you will lose your progress from other games you played. The next time you open your game, it will show you a guide native screen to create new account.

Still the small welcome native pop up doesn't show, not sure if it only doesn't show for local builds. I hope it will be fixed soon.

**Comment by Oninou** [comment]
I'm actually on v11.01 and i'm having the same issue, where the authentication is working but there is never the pop up telling the player that the authentication worked

**Comment by userqt** [comment]
Yep, I am still waiting for resolution of this issue. Any news? I am still checking this on internal or closed test. Haven't released it yet. I hope is should have nothing to do with whether the game is live or for test.

**Comment by AKey001** [comment]
I have watched this thread for some time because it was also not working in my project with v11.01. But today it worked without changing something inside the unity project.
So I think it was a problem within the Google play Games App, which is responsible for these popups. I updated it on my test device and now it works. So I think they changed something there. Be sure that this app is installed with the latest version.

**Comment by userqt** [comment]
[AKey001](https://github.com/AKey001) Oh man, I almost lost hope on this one. There was no update showing for the Google Play Games app. Even after clear cache and data for the playstore app. So based on your comment, I just uninstalled the Google Play Games App and installed. Then tried it with my app, and yep the message was there.

Thanks a lot.
